docker/run_tests.sh

```diff
@@ -38,15 +37,13 @@ PYTHON="$(which python)"
 
 echo ">>> wait for services to be ready"
 urlwait "${DATABASE_URL}" 10
-urlwait "http://${PUBSUB_EMULATOR_HOST}" 10
 urlwait "${ELASTICSEARCH_URL}" 10
 urlwait "${S3_ENDPOINT_URL}" 10
 urlwait "${SQS_ENDPOINT_URL}" 10
 
-echo ">>> build pubsub things and db things"
-# Clear Pub/Sub for tests
-./socorro-cmd pubsub delete
-./socorro-cmd pubsub create
+echo ">>> build sqs things and db things"
+# Clear SQS for tests
+./socorro-cmd sqs delete-all
```

Comment:
The tests are in charge of creating any queues they need. This script needs to clean the slate so tests aren't affected by previous runs.